recipe/bld.bat

```diff
@@ -1,22 +1,42 @@
-robocopy . "%PREFIX%" /S /XF bld.bat
-if errorlevel NEQ 1 exit 1
+set "PREFIX_NSIS=%PREFIX%\NSIS"
+robocopy . "%PREFIX_NSIS%" /S /XF bld.bat Docs Examples
+if errorlevel 1 exit 1
```

Comment:
Yeah I have no idea why we had `NEQ` before. Was worried it was some bizarro `robocopy` workaround. Thanks for fixing it.